DataManagementSystem/DB/FileCatalogComponents/DirectoryMetadata.py

```diff
@@ -604,9 +626,9 @@ def findDirIDsByMetadata(self, queryDict, path, credDict):
       first = True
       for meta, value in finalMetaDict.items():
         if value == "Missing":
-          result = self.__findSubdirMissingMeta(meta, pathSelection)
+          result = self.__findSubdirMissingMeta(fqmeta, pathSelection)
         else:
-          result = self.__findSubdirByMeta(meta, value, pathSelection)
+          result = self.__findSubdirByMeta(fqmeta, value, pathSelection)
```

Comment:
Should this still be meta, given that this comes from the loop variable four lines above?

Reply:
as above.